templater/jobs/presubmit/eks-distro-build-tooling/eks-distro-base-presubmits-java-2022.yaml

```diff
@@ -0,0 +1,37 @@
+jobName: eks-distro-base-tooling-presubmits-csi-2022
```

Comment:
This should be change to a Java-based name. Same for below

Reply:
ah yeah